ngraph/python/tests/test_frontend/test_frontend_onnx.py

```diff
@@ -95,3 +95,12 @@ def test_decode_and_convert():
     b = np.array([[2, 3], [4, 5]], dtype=np.float32)
     expected = np.array([[1.5, 5], [10.5, 18]], dtype=np.float32)
     run_function(decoded_function, a, b, expected=[expected])
+
+
+def test_load_by_model():
+    skip_if_onnx_frontend_is_disabled()
+
+    assert fem.load_by_model(onnx_model_filename)
+    assert fem.load_by_model("test.prototxt")
```

Comment:
we should check that such model is really converted

Reply:
right, done